packages/playwright/src/isomorphic/testServerInterface.ts

```diff
@@ -69,15 +69,14 @@ export interface TestServerInterface {
     files: string[];
   }): Promise<{ testFiles: string[]; errors?: reporterTypes.TestError[]; }>;
 
-  stop(): Promise<void>;
+  stopTests(): Promise<void>;
 
   closeGracefully(): Promise<void>;
 }
 
 export interface TestServerInterfaceEvents {
   onClose: Event<void>;
-  onListReport: Event<any>;
-  onTestReport: Event<any>;
+  onReport: Event<any>;
```

Comment:
Let's comment this fires with each TeleEmitter event in response to the `runTests` command, but not `listTests`.